build_tools/cmake/iree_cc_binary.cmake

```
@@ -192,4 +192,16 @@ function(iree_cc_binary)
       )
     endif()
   endif()
+
+  # Set up Info.plist properties when building macOS/iOS app bundles.
+  get_target_property(APPLE_BUNDLE ${_NAME} MACOSX_BUNDLE)
+  if (APPLE_BUNDLE)
+    set_target_properties(${_NAME} PROPERTIES
+      MACOSX_BUNDLE_GUI_IDENTIFIER "com.google.iree.${_RULE_NAME}"
+      # These are just placeholder version numbers until we define proper
+      # version scheme and support.
+      MACOSX_BUNDLE_BUNDLE_VERSION 0.1
+      MACOSX_BUNDLE_SHORT_VERSION_STRING 0.1
+      MACOSX_BUNDLE_LONG_VERSION_STRING 0.1)
```

Should we also set `MACOSX_BUNDLE_BUNDLE_NAME` and `MACOSX_BUNDLE_COPYRIGHT`? https://cmake.org/cmake/help/latest/prop_tgt/MACOSX_BUNDLE_INFO_PLIST.html

```suggestion
    set_target_properties(${_NAME} PROPERTIES
      MACOSX_BUNDLE_BUNDLE_NAME "${_RULE_NAME}"
      MACOSX_BUNDLE_GUI_IDENTIFIER "com.google.iree.${_RULE_NAME}"
      MACOSX_BUNDLE_COPYRIGHT "Copyright © 2023 The IREE Authors"
      # These are just placeholder version numbers until we define proper
      # version scheme and support.
      MACOSX_BUNDLE_BUNDLE_VERSION 0.1
      MACOSX_BUNDLE_SHORT_VERSION_STRING 0.1
      MACOSX_BUNDLE_LONG_VERSION_STRING 0.1)
```

---

Yes! Thanks. :D